### DISCUSSION TITLE
Let Renovate open dependency dashboard only on failed status checks not on pending

### DISCUSSION THREAD

**Discussion by Alexander-Betz** [renovatebot/renovate#15379] [Request Help] [answered]
I want to get notifications every time Renovate opens the dependency dashboard issue in a repository.

I only want to get notified when new PRs are created which happens in my config
- for major updates or 
- when Renovate can't automerge from a branch because of failing tests

As I understand it Renovate lists dependency updates under the "Pending Status Checks" section if tests failed or if tests are running because from what I could observe:
Renovate run 1: finds a new dependency update, immediately opens the dependency dashboard issue and lists the dependency update under "Pending Status Checks" while the tests are running.
Renovate run 2: Automerges the update if the tests passed and closes the issue again.

This results in many notifications for no reason. I want Renovate to wait and only open the issue if the tests failed not open it because there are running tests.

How can I do that?  Thank you in advance for any help.

My `config.js`:
```
module.exports = {
  "extends": [
    ":autodetectPinVersions",
    ":automergeBranch",
    ":automergeMinor",
    ":automergeRequireAllStatusChecks",
    ":combinePatchMinorReleases",
    ":dependencyDashboard",
    ":enableVulnerabilityAlerts",
    ":ignoreModulesAndTests",
    ":ignoreUnstable",
    ":label(renovate)",
    ":prNotPending",
    ":renovatePrefix",
    ":separateMultipleMajorReleases",
    ":updateNotScheduled",
    "group:monorepos",
    "group:recommended",
    "workarounds:all"
  ],
  onboarding: true,
  commitMessagePrefix: '[Renovate]',
  dependencyDashboardAutoclose: true,
  platform: 'github',
  requireConfig: false,
  schedule: null,
  reuseExistingBranch: true,
  updateLockFiles: true,
  transitiveRemediation: true
};
```

**Comment by rarkins** [comment, marked as answer]
I recommend that you disable autoclosing of the issue

**Reply by Alexander-Betz** [reply]
I tried it out today and disabled the autoclosing. But now the issue remains open and Renovate doesn't close it anymore. I want Renovate to close it if nothing is pending.
![image](https://user-images.githubusercontent.com/70712913/166213214-8bd7595e-d91c-4369-ac73-e9c3267e6da8.png)

I still want to get notifications every time the issue is reopened for any pending PRs (like I mentioned for major updates or failing tests).

I want to disable that Renovate reopens the issue just because a test is running for a new detected dependency update, like it was here the case now today. 
I'm running Renovate with the github action. 1st Renovate run detected the update and my tests started running. Renovate reopens the issue just because the test is running:
![image](https://user-images.githubusercontent.com/70712913/166214992-71fcb053-0618-4891-ad60-383c7cfe7594.png)

2nd Renovate run started running after the tests were complete (in this case passed), automerged the dependency and removed it from the list again
![image](https://user-images.githubusercontent.com/70712913/166215656-28fc2832-793d-4597-8a77-c5de8f7eff72.png)

**Reply by rarkins** [reply]
I don't think that the dashboard functionality fits your use case. But unfortunately I also don't think your use case fits Renovate well - the tradeoff of complication of settings won't be worth it. You'll need to decide whether to disable it permanently or keep it open.

**Reply by Alexander-Betz** [reply]
Okay, I understand. Thank you for the answer.
